### ISSUE TITLE
Prevent clear text passwords in process parameters also for WinSCP

### ISSUE THREAD

**Issue by WinSSHTerm** [WinSSHTerm/WinSSHTerm#10]
As noted [here](https://github.com/mRemoteNG/mRemoteNG/issues/304) (or [here](https://github.com/mRemoteNG/mRemoteNG/issues/181#issuecomment-255402193)), when opening a session with PuTTY (plink, WinSCP), the password will be visible in the process parameters. You can easily see those parameters by e.g. this PowerShell command
```
Get-CimInstance Win32_Process -Filter "name = 'winscp.exe' or name = 'putty.exe' or name = 'plink.exe'"| select CommandLine | Format-Table -wrap -AutoSize
```
**This is a security issue.**

WinSSHTerm already uses PuTTY's new feature `-pwfile` since version 2.23.1 to prevent passwords in clear text. Now WinSCP 6 is released with the new feature `/passwordsfromfiles`, so WinSSHTerm's implementation should be adapted to prevent passwords in clear text for WinSCP too.

**Comment by WinSSHTerm** [comment]
This was fixed in version 2.27.0

**Comment by WinSSHTerm** [comment]
Sample output for the PowerShell command (WinSSHTerm 2.27.0):
![process_arguments](https://github.com/WinSSHTerm/WinSSHTerm/assets/59282470/a3e70eea-24fa-4740-8982-c493d7489443)
